### PR TITLE
feat: add get_proposal_status

### LINKED ISSUE #524: `getProposalStatus`

### PR DESCRIPTION
**Pull request by pscott**
Closes #524

### CODE REVIEW

File: starknet/src/execution_strategies/simple_quorum.cairo [hunk 1]
```diff
@@ -67,3 +67,6 @@ mod SimpleQuorumExecutionStrategy {
         votes_for > votes_against
     }
 }
+// TODO: add unit tests for get_proposal_status
```
Comment (Orland0x) [inline starknet/src/execution_strategies/simple_quorum.cairo]: did you plan to add these on this PR?

Comment (pscott) [reply starknet/src/execution_strategies/simple_quorum.cairo]: I wasn't planning on but now that I think about it it does make more sense yeah. I'm on it.

Comment (pscott) [reply starknet/src/execution_strategies/simple_quorum.cairo]: 29519403023dfe7d2c94e1b61103ea97ba961e9e

**Review by Orland0x — APPROVED** [review]
uTACK

**Pull request merged**